source/app/components/EntitiesCheckbox/EntitiesCheckbox.js

```diff
@@ -113,7 +110,7 @@ export default function EntitiesCheckbox({
                       onSwitchPage(pageNumber)
                     }}
                   >
-                    Redact
```

Comment:
Are we looking to change the terms around redaction? Or are we no longer redacting all terms under a specific entity?

Reply:
This button should redact all terms of the entity it's next to so "Redact all" is a bit clearer than "Redact".